### PR TITLE
Integration login page with back-end api

### PR DESCRIPTION
**Pull request by lemonshark05**
Integration login page with back-end api

https://github.com/user-attachments/assets/88c08327-9eef-4ee5-b9eb-d980139b8798

https://github.com/user-attachments/assets/b51dafd4-63e0-4ef0-89f6-fb4ba91b8a6d

### CODE REVIEW

**Review by zhunan123 — APPROVED** [review]
looks good to me !

File: frontend/src/Login.js [hunk 1]
```diff
@@ -1,69 +1,144 @@
...
+            setPasswordError('Password must be at least 6 characters long.');
+            return;
         } else {
-            setError('');
-            // submission logic here
+            setPasswordError('');
+        }
+
+        // Define API endpoint from environment variable
+        const apiUrl = process.env.REACT_APP_API_BASE_URL + '/auth/login';
```
[the remark this answers is not in the document]
Comment (lemonshark05) [reply frontend/src/Login.js:43]: In front end .env file, add this line:
'''
REACT_APP_API_BASE_URL=http://localhost:8000
'''

File: frontend/src/Home.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (lemonshark05) [reply frontend/src/Home.js]: I delete the Home.js now

File: frontend/src/App.js [hunk 3]
```diff
@@ -18,6 +19,7 @@ function App() {
       <div className="App">
         <NavigationBar />
         <Routes>
+          <Route path="/home" element={<Home />} />
```
[the remark this answers is not in the document]
Comment (lemonshark05) [reply frontend/src/App.js]: I delete the route now.

**Pull request merged**